api/client/client.go

```diff
@@ -1319,6 +1319,52 @@ func (c *Client) DeleteAllNodes(ctx context.Context, namespace string) error {
 	return trail.FromGRPC(err)
 }
 
+// StreamSessionEvents streams audit events from a given session recording.
+func (c *Client) StreamSessionEvents(ctx context.Context, sessionID string, startIndex int) (chan events.AuditEvent, chan error) {
+	request := &proto.StreamSessionEventsRequest{
+		SessionID:  sessionID,
+		StartIndex: int32(startIndex),
+	}
+
+	ch := make(chan events.AuditEvent)
+
+	stream, err := c.grpc.StreamSessionEvents(ctx, request)
+	if err != nil {
+		close(ch)
+		e := make(chan error, 1)
+		e <- trace.Wrap(trail.FromGRPC(err))
+		return ch, e
```

Comment:
Here as well, if the caller just selects on both (as grpcserver.go does) they may not see the error

Reply:
Resolved

Reply:
I don't think you changed this one either, one more commit to push?